### PR TITLE
Add day in workspace-usage endpoint

### PR DESCRIPTION
**Pull request by albandum**
## Description

Some users want day-based granularity in the workspace-usage endpoint. 
Also piggy-backing to make sure the docs check in husky runs an npm format after building the swagger file.

### CODE REVIEW

File: front/pages/api/v1/w/[wId]/workspace-usage.ts [hunk 1]
```diff
@@ -26,22 +26,22 @@ export const usageTables = [
 ];
 type usageTableType = (typeof usageTables)[number];
 
-const MonthSchema = t.refinement(
+const DateSchema = t.refinement(
   t.string,
-  (s): s is string => /^\d{4}-(0[1-9]|1[0-2])$/.test(s),
-  "YYYY-MM"
+  (s): s is string => /^\d{4}-(0[1-9]|1[0-2])(-([0-2]\d|3[01]))?$/.test(s),
```
Comment (JulesBelveze) [inline front/pages/api/v1/w/[wId]/workspace-usage.ts:31]: `nit`: me hating long regex, could we do something like?
```
  (s): s is string => 
    /^\d{4}-(0[1-9]|1[0-2])$/.test(s) || 
    /^\d{4}-(0[1-9]|1[0-2])-([0-2]\d|3[01])$/.test(s),
```

Comment (JulesBelveze) [reply front/pages/api/v1/w/[wId]/workspace-usage.ts:31]: Your call tbh, it's not the craziest regex either..

**Review by JulesBelveze — APPROVED** [review]
Thx 🙏🏼

File: front/pages/api/v1/w/[wId]/workspace-usage.ts (hunk 1 above)
Comment (albandum) [reply front/pages/api/v1/w/[wId]/workspace-usage.ts:31]: Haha yes it doesn't look a lot less complicated :P

**Pull request merged**